docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1268,180 +1268,193 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
 
 [NOTE]
 ====
-If the resource name does not match the application name (or the overridden name) instead of reusing the resource a new one will be added. Same goes for the container.
-If the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
+* When the resource name does not match the application name (or the overridden name) instead of reusing the resource a new one will be added. Same goes for the container.
+
+* When the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
 ====
 
 == Service Binding [[service_binding]]
 
-Quarkus supports binding services to applications via the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
-Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing
-applications running in appropriately configured Kubernetes clusters to consume services (such as a Database or a Broker) without the need for user configuration.
+Generally, OpenShift applications and services, also referred to as deployable workloads, need to be connected to other services for retrieving additional information such as service URLs or credentials.
```

Comment:
```
Generally, OpenShift applications and services, also referred to as deployable workloads, need to be connected to other services for retrieving additional information such as service URLs or credentials.
The required communication for obtaining this information is managed by the Service Binding Operator, which also determines:
```
I would revert this part to the original.

Reply:
I agree, this new section seems to muddy the waters

Reply:
Ok, Will do